master.yaml

```diff
@@ -9,26 +9,42 @@
 tests:
   - name: pod_kill
     test_tags: ['mysql', 'pumba']
-    location: ./apps/percona/tests/mysql_data_persistence/run_litmus_test.yaml
-    test_params:
-      - name: DB_SERVER_IP
-        kind: env
-        type: string
+    specPath: ./apps/percona/tests/mysql_data_persistence/run_litmus_test.yaml
+    patch:
+    - name: 'Server IP'
+      desc: ''
+      kind: env
+      type: string
+      op: 'add'
```

Comment:
should it be add or replace?

Reply:
I have checked `apps/percona/tests/mysql_data_persistence/run_litmus_test.yaml` (`specPath`) there is no environment variable named `DB_SERVER_IP` in `ansibletest` container. So, I think it will be `'add'` instead of `'replace'`.

Reply:
@ksatchit Then we should update `master.yaml` accordingly in upcoming PRs. Also we can use any of the verbs mentioned in the PR description anytime if we feel so.